Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ however, it has to be formatted properly to pass verification tests.
 ### Fixed
 
 - Fixed pairing devices to existing `InputUser`s potentially corrupting list of paired devices from other `InputUser`s ([case 1327628](https://issuetracker.unity3d.com/issues/input-system-devices-are-reassigned-to-the-wrong-users-after-adding-a-new-device)).
+- Fixed `InputControlPathDrawer` drawing every entry in a collection of `InputControl`s as the first entry.
```

Comment:
`InputControlPathDrawer` is an internal class. Better to describe in terms of what broke when you did what. Maybe something like
```
* Fixed inspector for arrays/lists of `[InputControl]` paths duplicating the value of the first element as the value of all others (...).
```

Or something like that.